### ISSUE TITLE
Create a wins form.

### ISSUE THREAD

**Issue by jbubar** [hackforla/website#1060]
### Overview
Create a wins form to replace the current google form following the [figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1209%3A0)
The ultimate goal is to have this form populate a google sheet which team managers can review and approve.

### Action Items
- [x] create an html form called something like wins-form.html
- [x] populate it with questions shown on the figma
- [x] name the columns something that makes sense to you and is inline with the code for the wins page

### Resources/Instructions
[figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1209%3A0)
[medium article about how to make a form submit to google sheet](https://medium.com/@dmccoy/how-to-submit-an-html-form-to-google-sheets-without-google-forms-b833952cc175)

**Comment by akibrhast** [comment]
## Update
- The current functionality of the wins form stands such that submitting the form populate the new google sheet linked below.
Logical implementation wise and functionality wise it's almost complete. Except for two form fields that I have yet to set up.
- I have spoken to @trtincher and he is willing to take over from me to implement the css changes needed to match the figma specs
- I have spoken to @jbubar and he said he will finish up implementing the refactoration of wins page. Which is almost done(except the see more functionality.)
- You can pull and continue working from this branch using the following commands.
```
git remote add akibr https://github.com/akibrhast/website
git checkout akibr/wins-works -b wins-works
```


### Progress
- Created the html form
- Refactored wins pages
- Created google [sheets ](https://docs.google.com/spreadsheets/d/1zkr_dEyiT-WTksUkVyL4jYQuYC5_YvJCLrUIfBf1CeE/edit?usp=sharing)with sensible column names 

### Blockers
- For this issue to be merged and finalized, I believe the new excel sheet needs to be approved by @ExperimentsInHonesty 

### Avaibality
- At most 3 hours for the weeks of 22nd February

### ETA
- 5th March

**Comment by jbubar** [comment]
Just an update. I think there is a small mistake on this git command shared above.
``` diff
- git remote add akibr git@https://github.com/akibrhast/website
+ git remote add akibr https://github.com/akibrhast/website
git checkout akibr/wins-works -b wins-works
```

**Comment by akibrhast** [comment]
## Progress
The requested action items has been completed. Except for the design parts.
@jbubar and @trtincher has taken over continuing the work from my branch and I will not be making any more commits to that branch. I believe this issue can be now be closed with a creation of a new issue that states 
>  style it to look like the figma (make it responsive so that it looks similar on mobile as it would on desktop)


## Blockers
None.

## Availability
Not available until 3rd March

## ETA
None.

**Comment by jbubar** [comment]
@akibrhast can you create a new issue to style the wins form with this as a dependency. ***they can start working on it once this is merged

**Comment by akibrhast** [comment]
@jbubar New issue created #1104 with this as dependenycy

**Comment by akibrhast** [comment]
@jbubar When do you think you will be able to make a pull request to close this issue?

**Comment by jbubar** [comment]
edit this dependency #1163 when this issue is closed.

**Comment by akibrhast** [comment]
## Progress
Complete. Waiting on the merge of the linked pull request

## Blockers
none

## Availability
evenings

## ETA
complete
